### ISSUE TITLE
firebase-ios-sdk.git has no Package.swift manifest for version 6.26.0

### ISSUE THREAD

**Issue by trevor-sonic** [firebase/firebase-ios-sdk#6497]
#### Environment

  * Xcode version: 12.0
  * Firebase SDK version: 6.26.0
  * Firebase Component: na
  * Component version: na
  * Installation method:  Swift Package Manager 

#### Steps to reproduce:
Xcode12 -> File->Add package dependency-> https://github.com/firebase/firebase-ios-sdk.git
It didn't download, gave message below.

```https://github.com/firebase/firebase-ios-sdk.git has no Package.swift manifest for version 6.26.0```

**Comment by paulb777** [comment]
The Swift Package Manager beta started with version 6.31.0.  

Usage details at https://github.com/firebase/firebase-ios-sdk/blob/master/SwiftPackageManager.md

**Comment by trevor-sonic** [comment]
Hi,
Setting the branch to : ```6.32-spm-beta``` allowed me to add Firebase into the project.
Thank you so much.
Now, I'm having the compile error message below. But I'm not sure if it is ```Firebase``` related.
I'll update here as soon as figure it out.

```
Undefined symbols for architecture x86_64:
  "_kFIREventLevelEnd", referenced from:
      MyProject.DBConfig.(logChanges in _70FD9EC2C32030944DDEE8806FED63D7)(new: Swift.UInt64, old: Swift.UInt64) -> () in DBConfig.o
```

**Comment by paulb777** [comment]
@trevor-sonic The undefined symbol linker error looks like #6425 which may be fixed in 6.33-spm-beta. Would you give that branch a try and let us know?

**Comment by trevor-sonic** [comment]
I tried ```6.33-spm-beta ``` project compiled, but ```import Firebase``` gives library not found error, so I removed again, I'll try later with next release.
